py3status/modules/xrandr_toggle.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Toggle monitor output from the status bar using xrandr.
+
+Configuration parameters:
+    cache_timeout: how often we refresh usage in seconds (default: 10s)
+    distext: custom text or symbol for disconnected status
```

Comment:
> I have often a hard time keeping the 80 characters limit per line

IMHO, generally it's a sign that you need to refactor the code as it's getting too complicated.
Also strictly speaking it's 79 chars.

When I first started using python I found it a pain with all the indentation and lines drifting off to the right.  But it taught me how to write cleaner nicer code after a couple of years.